### PR TITLE
Preparing for release of 0.6.1 of ms_rest & ms_rest_azure

### PR DESCRIPTION
**Pull request by vishrutshah**
This is part 3 of the feature work https://github.com/Azure/azure-sdk-for-ruby/issues/517

### CODE REVIEW

**Comment by vishrutshah** [comment]
@veronicagg & @sarangan12 Feel free to take a look when you get a chance. Thanks!

File: runtime/ms_rest/README.md [hunk 1]
```diff
@@ -51,6 +51,15 @@ Reference it in the gemfile and also add this line to your client's gemspec file
 spec.add_runtime_dependency 'ms_rest', '~> 0.6.0'
```
Comment (veronicagg) [inline runtime/ms_rest/README.md]: should we update this one too?

File: runtime/ms_rest_azure/README.md [hunk 2]
```diff
@@ -40,6 +40,15 @@ Reference it in the gemfile and also add this line to your client's gemspec file
 spec.add_runtime_dependency 'ms_rest_azure', '~> 0.6.0'
```
Comment (veronicagg) [inline runtime/ms_rest_azure/README.md]: should we update this version too?

File: runtime/ms_rest/README.md [hunk 3]
```diff
@@ -51,6 +51,15 @@ Reference it in the gemfile and also add this line to your client's gemspec file
...
+# Utilizing Telemetry Extension in your SDK
+
+We encourage the customer of ms_rest library to provide information about their product sent via telemetry extension point as below:
+
+```ruby
+additional_user_agent_information = 'fog-azure-rm/0.2.0' # Refer https://github.com/Azure/azure-sdk-for-ruby/issues/517 for more info.
+service_client.add_user_agent_information(additional_user_agent_information)
+```
+
 Don't forget to correct the version.
```
Comment (veronicagg) [inline runtime/ms_rest/README.md]: is this sentence supposed to be before the Telemetry section?

Comment (vishrutshah) [inline runtime/ms_rest/README.md]: Yes good. Good catch. I'll update it along with the dates in changelog.

File: runtime/ms_rest_azure/README.md (hunk 2 above)
Comment (vishrutshah) [inline runtime/ms_rest_azure/README.md]: Yup. Fixed

**Comment by vishrutshah** [comment]
Thanks for the review!!

**Pull request merged**